test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-stock-orderpoint-manual-procurement @ git+https://github.com/trobz/stock-logistics-warehouse@14.0-mig-stock_orderpoint_manual_procurement#subdirectory=setup/stock_orderpoint_manual_procurement
```

Comment:
The dependency has been merged: https://github.com/OCA/stock-logistics-warehouse/pull/1054

Reply:
ok @sebalix , I pushed the update